website/static/js/fangorn.js

```diff
@@ -46,6 +46,26 @@ var ENTER_KEY = 13;
 
 var ICON_PATH = '/static/img/hgrid/fatcowicons/';
 
+// Add Authorization Bearer Token to request
+var oldmrequest = m.request;
+m.request = function () {
```

Comment:
It might make sense to put this shim in `base-page.js`, just in case other modules need this.